src/components/app/ProfileViewer.js

```diff
@@ -69,6 +74,13 @@ class ProfileViewer extends PureComponent<Props> {
           profileViewerWrapperBackground: hasSanitizedProfile,
         })}
       >
+        {icons.map(({ className, icon }) => (
```

Comment:
Optional, mainly just wondering: I wonder how large the icons list will get. It seems like wrapping this map into it's own PureComponent means that we would only re-compute these components when the icons actually changed. Here it's re-computing it for any state invalidation. Although, maybe this is just a minimal update and not worth optimizing.

Reply:
We don't add the icons until we need them, and that keeps the icons array pretty small. For example in the profile I posted to the first comment, the array length is 3. But yeah, I agree that we can add a pure component and make it chached. Will do as a follow-up :)